<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-H5KK288" failures="0" tests="3" name="com.ecommerce.tests.FunctionalFlowTests" time="46.227" errors="1" timestamp="18 Mar 2020 15:18:19 GMT">
  <testcase classname="com.ecommerce.tests.FunctionalFlowTests" name="loginTest" time="2.162"/>
  <testcase classname="com.ecommerce.tests.FunctionalFlowTests" name="addDeleteProductsCheckout" time="42.131"/>
  <testcase classname="com.ecommerce.tests.FunctionalFlowTests" name="filterProductTest" time="1.934">
    <error message="expected [3] but found [7]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [3] but found [7]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:370)
	at org.testng.Assert.assertEquals(Assert.java:380)
	at com.ecommerce.tests.FunctionalFlowTests.filterProductTest(FunctionalFlowTests.java:57)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:882)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1189)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:830)
]]>
    </error>
  </testcase> <!-- filterProductTest -->
</testsuite> <!-- com.ecommerce.tests.FunctionalFlowTests -->
